build.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+mkdir build && cd build && ^
+cmake -G "Visual Studio 15 Win64" ../ && "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64 && ^
+MSBuild xLearn.sln /p:Configuration=Release && ^
```

Comment:
What happen if a user doesn't install his visual-studio in C: ?

Reply:
emm, so, we should build .whl for windows, and users can install xLearn by pip

Reply:
I build binary with AppVeyor. This is the file url https://ci.appveyor.com/project/etveritas/xlearn/branch/etveritas-xlearn/artifacts.
we can add xLearn to  travis or  AppVeyor for Continuous Integration.